sunpy/physics/limb_darkening.py

```diff
@@ -0,0 +1,82 @@
+import copy as c
+import sunpy.map
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+__author__ = ["Norbert Gyenge"]
+
+
+def limbdark(observation, limb_cut=True):
+    """
+    Calculates the limb darkening function for a specified wavelength
+    and removes any limb darkening effect from the input image.
+    A 5th order polynomial is fit to the limb darkening constants obtained from
+    Astrophysical Quantities by Allan. Only in the wavelength
+    range between 4000 and 15000 A.
+    Parameters
+    ----------
+        observation - SunPy Map object
```

Comment:
remove indentation and format like:
```python
observation: `sunpy.map.GenericMap`.
    Input map object
limb_cut: `bool`
    If `True` background (r>1) values will be repleaced by np.nan.
```

Reply:
I think the space before the colon is important for sphinx to render properly i.e.
```python
observation : `sunpy.map.GenericMap`
```